api/query/cache/index/filter.go

```diff
@@ -120,6 +126,15 @@ func (tnp TestNamePattern) Filter(t TestID) bool {
 	return strings.Contains(name, tnp.q.Pattern)
 }
 
+// Filter interprets a SubtestNamePattern as a filter function over TestIDs.
+func (tnp SubtestNamePattern) Filter(t TestID) bool {
+	_, subtest, err := tnp.tests.GetName(t)
```

Comment:
GetName(TestID) - does it mean each testID associates with a unique(testname, subtestname) combination?

Reply:
Yes